pkg/api/orchestrators.go

```diff
@@ -53,21 +60,31 @@ func validate(orchestrator, version string) (string, error) {
 	return "", nil
 }
 
-func isVersionSupported(csOrch *OrchestratorProfile) bool {
+func isVersionSupported(csOrch *OrchestratorProfile, onAzureStack bool) bool {
 	supported := false
-	for _, version := range versionsMap[csOrch.OrchestratorType] {
+	if onAzureStack {
+		for _, version := range versionsMapAzureStack[csOrch.OrchestratorType] {
 
-		if version == csOrch.OrchestratorVersion {
-			supported = true
-			break
+			if version == csOrch.OrchestratorVersion {
+				supported = true
+				break
+			}
+		}
+	} else {
+		for _, version := range versionsMap[csOrch.OrchestratorType] {
+
+			if version == csOrch.OrchestratorVersion {
+				supported = true
+				break
+			}
 		}
 	}
 	return supported
 }
```

Comment:
This may be easier to read

```
supported := false
versions := versionsMap[csOrch.OrchestratorType]
if onAzureStack {
	versions := versionsMapAzureStack[csOrch.OrchestratorType]
} 
for _, version := range versions {
	if version == csOrch.OrchestratorVersion {
		supported = true
		break
	}
}
return supported
```

Reply:
addressed